waltid-wallet-api/src/main/kotlin/id/walt/webwallet/db/models/WalletCategory.kt

```diff
@@ -0,0 +1,21 @@
+package id.walt.webwallet.db.models
+
+import kotlinx.serialization.Serializable
+import org.jetbrains.exposed.sql.ResultRow
+import org.jetbrains.exposed.sql.Table
+
+object WalletCategory : Table("category") {
```

Comment:
I like this modeling approach, but it will also require a unique index on `(wallet, name)` and this means that other similar tables would require the same update (e.g.` WalletKeys`) triggering a chain of other updates (which I consider out of scope for this task). Maybe we can have a separate task for updating the db models to surrogate index, where appropriate.